README.rst

```diff
@@ -71,6 +71,16 @@ test re-run is launched:
 
    $ pytest --reruns 5 --reruns-delay 1
 
+To use a random backoff to vary the delay, use the ``--reruns-delay-backoff``
+command line options. This causes the delay to be picked randomly between 0
+and a maximum value that doubles on every failure (starting value is set by
```

Comment:
Instead of doubling, did you consider allowing the user to specify the base for the exponent rather than hard-coding `2`? And did you consider allowing the user to specify a limit so the exponential progression doesn't get too crazy?

Reply:
With your other changes, "doubles" can now be generalised to "increases".

Reply:
Done